pkg/resourceinterpreter/default/thirdparty/resourcecustomizations/argoproj.io/v1alpha1/Workflow/customizations.yaml

```diff
@@ -0,0 +1,176 @@
+apiVersion: config.karmada.io/v1alpha1
+kind: ResourceInterpreterCustomization
+metadata:
+  name: declarative-configuration-workflow
+spec:
+  target:
+    apiVersion: argoproj.io/v1alpha1
+    kind: Workflow
+  customizations:
+    replicaResource:
+      luaScript: >
+        local kube = require("kube")
+        function GetReplicas(obj)
+          replica = 1
+          if obj.spec.parallelism ~= nil then
+            replica = obj.spec.parallelism
+          end
+          podTemplateSpec = {}
+          podTemplateSpec.spec = {}
+          podTemplateSpec.spec.affinity = obj.spec.affinity
+          podTemplateSpec.spec.nodeSelector = obj.spec.nodeSelector
+          podTemplateSpec.spec.tolerations = obj.spec.tolerations
+          requirement = kube.accuratePodRequirements(podTemplateSpec)
+          return replica, requirement
+        end
+    replicaRevision:
+      luaScript: >
+        function ReviseReplica(obj, replica)
+          obj.spec.parallelism = replica
+          return obj
+        end
+    healthInterpretation:
+      luaScript: >
+        function InterpretHealth(observedObj)
+          if observedObj.status == nil then
+            return false
+          end
+          if observedObj.status.phase == nil or observedObj.status.phase == '' or observedObj.status.phase == 'Failed' or observedObj.status.failed == 'Error' then
+            return false
+          end
+          return true
+        end
+    retention:
+      luaScript: >
+        function Retain(desiredObj, observedObj)
+          if observedObj.spec.suspend ~= nil then
+            desiredObj.spec.suspend = observedObj.spec.suspend 
+          end
+          if observedObj.status ~= nil then
+            desiredObj.status = observedObj.status
+          end
```

Comment:
Based on the discussion at the community meeting, it was decided to retain the entire `.status`. And since workflow does not support subresource of status, the `AggregateStatus` operation was not implemented. 
/cc @Poor12 @chaunceyjiang

Reply:
Thanks.